news/2023/2023-02-10-osumonthly-7.md

```diff
@@ -0,0 +1,263 @@
+
+<iframe width="100%" height="315" src="https://www.youtube.com/embed/P6Y9B1pyIcw" frameborder="0" allowfullscreen></iframe>
+
+As osu!(lazer) is getting closer and closer to what the community would consider a "full release", more people have been getting engaged with the game, submitting pull requests and reporting bugs. This has resulted in (along other things) many cool features being added, such as the dynamic BPM counter, various versions of the judgement counters and a new song progress display bar. No longer will live streamers and players alike have to download and use 3rd party plugins in order to show these details, a common trend seen in osu!(lazer)'s development.
+
+As you are probably aware of, in December a change was made that allows for users to obtain performance points for playing maps in osu!(lazer), albeit still having separate rankings from the current game. The launch was not ideal, but there was still a huge increase in player numbers — and most importantly, a lot of feedback was given to the team. That being said, as mentioned in the video digest of the latest update, peppy is communicating with various community members in order to discuss what balance changes should be made to the game, so be on the lookout as to what future changes might bring.
+
+January has been full of small changes that add up to a big changelog, so if you are interested in the nitty-gritty of everything that's been changed or updated, feel free to either look at the video above or [read the full changelog](https://osu.ppy.sh/home/changelog/lazer/2023.121.0)!
+
+A huge feature of osu!(lazer) that's a complete game changer for people coming over from osu!(stable) is the fact that you do not need to have an osu!supporter tag in order to download maps directly, you can now do it directly from the in-game browser! You might also notice the fact that once a map is downloaded, it is automatically imported in the background (no need to press F5 anymore), and the in-game browser will show you which maps you already have!
+
+![](https://assets.ppy.sh/media/osu-monthly/2023-01/feature-showcase-jan23.mp4)
+
+## Community recap
+
+### osu!stream available on Android
+
+Back in the ancient days of 2011, [osu!stream](/wiki/osu!stream) was released onto the [App Store](https://apps.apple.com/us/app/osu-stream/id436952197) on iOS, and it has been maintained up until 2020. For many years people have been asking for an Android release, but ever since the start peppy knew this would not be an enjoyable experience due to audio latency issues that were deemed inevitable at the time.
+
+Well... 12 years later and [ThePooN](https://osu.ppy.sh/users/718454) along with various open-source contributors finally made it happen! You can now [download osu!stream](https://play.google.com/store/apps/details?id=sh.ppy.osustream&pli=1) on your Android phone and enjoy custom maps along with mechanics only found on osu!stream (one example being the hold hit circle).
```

Comment:
I'm not sure "custom maps" makes sense here. It sounds like you are able to load beatmaps or something.

I'd word it something like "enjoy unique beatmaps with mechanics only found.."

Reply:
Thanks, amended in https://github.com/ppy/osu-wiki/pull/8776/commits/214bac3dc4449a3bec9adfd4a7cfc126713e29ef